airbyte-integrations/connectors/source-stripe/source_stripe/authenticator.py

```diff
@@ -0,0 +1,20 @@
+from datetime import datetime, timedelta
+from typing import Tuple, Union
+from airbyte_cdk.sources.streams.http.requests_native_auth import Oauth2Authenticator
+
+
+class StripeOauth2Authenticator(Oauth2Authenticator):
```

Comment:
can we use the [existing DeclarativeOauth2Authenticator as-is](https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/declarative/auth/oauth.py#L18)? this doesn't seem to need a custom class

Reply:
Hi @girarda, thanks for the review. I will be working through your comments today. 

On this one, stripe does not return a field with an expires at value. The documentation states that all access tokens last 1 hour. Because of this, the AbstractOauth2Authenticator fails on the following line. 
https://github.com/tcboles/airbyte/blob/4f641a798d15cbbcd0981bd41ae4fa5bbe54ee7d/airbyte-cdk/python/airbyte_cdk/sources/streams/http/requests_native_auth/abstract_oauth.py#L152